commodore/cli/catalog.py

```diff
@@ -28,8 +32,26 @@ def clean(config: Config, verbose):
     clean_working_tree(config)
 
 
+def _complete_clusters(ctx: click.Context, _, incomplete: str) -> list[str]:
+    config = Config(Path("."))
+    config.api_url = ctx.params["api_url"]
+    config.api_token = ctx.params["api_token"]
+    config.oidc_client = ctx.params["oidc_client"]
+    config.oidc_discovery_url = ctx.params["oidc_discovery_url"]
+
+    try:
+        if config.api_token is None:
+            login(config)
```

Comment:
I'm fine with fixing that by just adding a note in the docs. Alternatively, we could just add a flag to suppress output from `login()` which we can use here, since we can only support autocompletion for local installs which should be able to open a browser tab anyway.

Reply:
I think suppressing output from `login` sounds like a good idea.

Reply:
> I think suppressing output from `login` sounds like a good idea.

Turns out `webbrowser.open()` tells you if it was able to open the link. So we can make the print conditional only when the page couldn't be opened automatically.

Reply:
That actually sounds like good idea in general

Reply:
I've created a follow-up PR (#747) to change login to only print when it can't open the URL, since this is something that should be visible in the changelog.